Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/architecture/Time.cpp

```diff
@@ -0,0 +1,36 @@
+#include "Time.hpp"
+
+
+Time* Time::instance = nullptr;
+float Time::deltaTime = 0.0f;
```

Comment:
Is it the constant you want to use everywhere after ?

Reply:
yes